sdk/storage/azure-storage-file-datalake/tests/perfstress_tests/append.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import uuid
+
+from azure_devtools.perfstress_tests import RandomStream, AsyncRandomStream
+
+from ._test_base import _FileSystemTest
+
+
+class AppendTest(_FileSystemTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        file_name = "filetest-" + str(uuid.uuid4())
+        self.file_client = self.fs_client.get_file_client(file_name)
+        self.async_file_client = self.async_fs_client.get_file_client(file_name)
+
+    async def setup(self):
+        await self.async_file_client.create_file()
+
+    def run_sync(self):
+        data = RandomStream(self.args.size)
+        self.file_client.append_data(
```

Comment:
Can appending to the same file over and over cause issues with service perf or correctness (e.g. is there a max file size)?

Reply:
I will check in with the Storage team to see if they can shed light on that

Reply:
There should be no problems. When you keep re-appending, you would be rewriting the appended data (i.e the last append wins - assuming same length and offset). As for the maximum size, its 4.75 TiB (100 MiB X 50,000 blocks) for version 2016-05-31 and later, and 195 GiB (4 MiB X 50,000 blocks) for all older versions (since a datalake file is a block blob).